<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>929</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>929</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1080</width>
    <height>929</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1080</width>
    <height>929</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Studio Babel</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/logo.png</normaloff>Images/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>1080</width>
     <height>929</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1080</width>
     <height>929</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1080</width>
     <height>929</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>1080</width>
     <height>929</height>
    </size>
   </property>
   <widget class="QLabel" name="_label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>929</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1080</width>
      <height>929</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1080</width>
      <height>929</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/background.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="_label_logo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>51</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="_label_bandeau">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>130</y>
      <width>791</width>
      <height>109</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>781</width>
      <height>109</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>805</width>
      <height>109</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>805</width>
      <height>109</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/bandeau.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="_label_backgroundBtnAdd">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>830</y>
      <width>271</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>271</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>281</width>
      <height>64</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/backgroundBtnAdd.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="_label_backgroundBtnCall">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>570</y>
      <width>771</width>
      <height>52</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/backgroundBtnCall.png</pixmap>
    </property>
   </widget>
   <widget class="QTextEdit" name="_text_chat">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>630</y>
      <width>771</width>
      <height>211</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>771</width>
      <height>211</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>771</width>
      <height>211</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>771</width>
      <height>211</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="_text_msg">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>850</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="_btnClose">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/btnClose.png</normaloff>Images/btnClose.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="_line_addContact">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>810</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>271</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="clearButtonEnabled" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="_btnAddContact">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>840</y>
      <width>43</width>
      <height>43</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>43</width>
      <height>43</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>43</width>
      <height>43</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>43</width>
      <height>43</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/btnAddContact.png</normaloff>Images/btnAddContact.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>43</width>
      <height>43</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="_btnInviteContact">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>580</y>
      <width>25</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/btnAddContact.png</normaloff>Images/btnAddContact.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="_btnMicro">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>580</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/btnMicro.png</normaloff>Images/btnMicro.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="_btnCam">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>580</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/btnCam.png</normaloff>Images/btnCam.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="_btnHangUp">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>580</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/btnHangUp.png</normaloff>Images/btnHangUp.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="_label_name">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Surname :</string>
    </property>
   </widget>
   <widget class="QLabel" name="_label_nickname">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="_label_PhoneNumberImg">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/btnPhoneNumber.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="_label_PhoneNumberValue">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>06 02 02 02 02</string>
    </property>
   </widget>
   <widget class="QLabel" name="_label_Localisation">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/btnLocalisation.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="_label_LocalisationValue">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Angers</string>
    </property>
   </widget>
   <widget class="QLabel" name="_label_Birthsday">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>150</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/btnBirthday.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="_label_BirthsdayValue">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>155</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>26 / 05 / 1992</string>
    </property>
   </widget>
   <widget class="QLabel" name="_label_NameValue">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Desmars</string>
    </property>
   </widget>
   <widget class="QLabel" name="_label_NickNameValue">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Napsters</string>
    </property>
   </widget>
   <widget class="QListWidget" name="_listContact">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>131</y>
      <width>271</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_Online">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/greenStatus.png</normaloff>Images/greenStatus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_Away">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/orangeStatus.png</normaloff>Images/orangeStatus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_Busy">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/redStatus.png</normaloff>Images/redStatus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="_label_Video">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>721</width>
      <height>251</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>721</width>
      <height>251</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>721</width>
      <height>251</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_label_StatusInfos">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>5</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>En ligne</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
